docs/courses/exercises2018.rst

```diff
@@ -0,0 +1,105 @@
+NEURON Course Exercises
+=======================
+
+These exercises are adapted from the 2018 NEURON summer course given in Atlanta, Georgia.
+
+1.
+    :ref:`intro_to_gui`
+
+    * Lipid bilayer model
+
+2.    
+    :ref:`interactive_modeling`
+
+    * Squid axon model
+
+3.    
+    The CellBuilder
+
+    * Constructing a ball and stick model, saving session files
+
+4.    
+    Using morphometric data
+
+    * The Import3D tool
+    
+5.
+    :ref:`neuron_scripting_exercises`
+    
+6.
+    :ref:`using_nmodl_files`
+
+    * Single compartment model with HHk mechanism
+
+7.    
+    Using ModelDB and ModelView
+
+    * Neuroinformatics tools for finding and understanding models
+
+8.    
+    :ref:`rxd_exercises`
+
+9.    
+    Specifying inhomogeneous channel distributions
+    with the CellBuilder
+
+10.    
+    :ref:`custom_initialization`
+    
+11.
+    Introduction to the linear circuit builder
+
+    * A two-electrode voltage clamp
+    
+12.
+    :ref:`state_and_parameter_discontinuities`
+    
+13.
+    Bulletin board parallelization
+
+    * Speed up embarrassingly parallel tasks
+    
+14.
+    HOC exercises
+
+    * Introduction to the HOC programming language
+    
+15.
+    :ref:`multithread_parallelization`
+    
+16.
+    Networks: discrete event simulations with artificial cells
+
+    * Introduction to the Network Builder
+    
+17.
+    Networks: continuous simulation of nets with biophysical model cells
+
+    * Network ready cells from the CellBuilder
+    
+18.
+    Hopfield-Brody network in Python
+    
+19.
+    MPI Parallelization
+    
+20.
+    :ref:`using_nsg_portal`
+    
+21.
+    :ref:`numerical_methods_exercises`
+    
+22.
+    Electrotonic Analysis
+
+.. toctree::
+    :hidden:
+
+    neuron_scripting_exercises.rst
```

Comment:
The `toctree` (table of contents tree) establishes the relative position of the documents in the documentation. 

If you click into `neuron_scripting_exercises`, you'll see at the top that it's part of the course exercises; if you click into the two new sections, you won't see this relationship... so please add them to the beginning of the `toctree` here.